static/styles/rendered_markdown.css

```diff
@@ -350,8 +357,9 @@
     .twitter-image img,
     .message_inline_image img,
     .message_inline_ref img {
-        height: auto;
-        max-height: 100%;
+        height: 100%;
```

Comment:
This is needed because we need to explicitly specify the image dimensions to have `object-fit` work; likely because internally `object-fit` needs to know the frame its targeting and images don't default to container dimensions.